src/main/java/com/team01/carsharingapp/mapper/PaymentMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.team01.carsharingapp.mapper;
+
+import com.team01.carsharingapp.config.MapperConfig;
+import com.team01.carsharingapp.dto.payment.PaymentDto;
+import com.team01.carsharingapp.dto.payment.PaymentRequestDto;
+import com.team01.carsharingapp.model.Payment;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+
+@Mapper(config = MapperConfig.class)
+public interface PaymentMapper {
+    PaymentDto toDto(Payment payment);
+
+    Payment toEntity(PaymentRequestDto paymentRequestDto);
+
+    @AfterMapping
+    default void setOrderId(@MappingTarget PaymentDto paymentDto, Payment payment) {
+        paymentDto.setRentalId(payment.getRental().getId());
+    }
```

Comment:
Usage @ Mapping(target="...", source="...") for easy set more preferable

Reply:
it doesn't work like that with a value like that. I don’t know how I can get the ID from Rental and set to dto, is there either an additional function needed or what I wrote earlier, like we did in the book store